README.md

```diff
@@ -330,7 +330,7 @@ InstrumentationSystem.bootstrap(MultiplexInstrument([FancyInstrument(), OtherFan
 <a name="passing-context-objects"></a>
 ### Context propagation, by explicit `LoggingContext` passing
 
-> `LoggingContext` naming has been carefully selected and it reflects the type's purpose and utility: It binds a [Swift Log `Logger`](https://github.com/apple/swift-log) with an associated distributed tracing [Baggage](https://github.com/apple/swift-distributed-tracing-baggage).
+> `LoggingContext` naming has been carefully selected and it reflects the type's purpose and utility: It binds a [Swift Log `Logger`](https://github.com/apple/swift-log) with an associated distributed tracing [Baggage](https://github.com/apple/swift-service-context).
```

Comment:
```suggestion
> `LoggingContext` naming has been carefully selected and it reflects the type's purpose and utility: It binds a [Swift Log `Logger`](https://github.com/apple/swift-log) with an associated service context [ServiceContext](https://github.com/apple/swift-service-context).
```

Reply:
I guess the intention was to embed type names in the sentence, so this should be better:
```suggestion
> `LoggingContext` naming has been carefully selected and it reflects the type's purpose and utility: It binds a [`Logger`](https://github.com/apple/swift-log) with an associated [`ServiceContext`](https://github.com/apple/swift-service-context).
```